jaeger/jaeger_install/rhbjaeger-removing.adoc

```diff
@@ -0,0 +1,25 @@
+[id="removing-jaeger"]
+= Removing Jaeger
+include::modules/jaeger-document-attributes.adoc[]
+:context: removing-jaeger
+
+toc::[]
+
+This process lets you remove Jaeger from a {product-title} cluster.
+
+.Prerequisites
+
+* Shut down any Jaeger pods
```

Comment:
These don't seem like prerequisites for the removal tasks? Seems more like goals of the tasks.